migrations/V121__Update_Precedence_Rules.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TYPE YES_NO_TYPE AS ENUM (
+  'YES',
+  'NO'
+  );
+
+ALTER TABLE plan_beneficiaries
+  ADD COLUMN precedence_rules_note ZERO_STRING;
+
+UPDATE plan_beneficiaries
+  SET precedence_rules_note = precedence_rules;
```

Comment:
Thanks @OddTomBrooks ,

One potential issue with this, if the row wasn't ever updated, this update will error because of the auditing trigger. The `modified_by` and `modified_dts` fields should be updated to ensure that there isn't an error. In other migrations we use the system account UUID and current_timestamp for these fields

Reply:
Ahhh I didn't think of this! This worked in local testing because I had to set the modified_by / modified_dts field when I manually set the precedence_rules. Thanks for pointing this out, I'll go find a migration addressing this and copy its process. Will have to think about the testing process.

Reply:
No worries! I sometimes do it with a couple of test cases with SQL. My DB management tool can save just some individual scripts for me, so I can run all the old migrations, then insert test data, then run the new one. It makes it a little less or a burden to test.

I'd probably suggest just having a record that isn't updated, (or perhaps add some filtering to the update statement, you don't really need to update records that don't have precedence rules, this will update everything currently). Still should probably set the modified properties to be safe, but the where clause would help also.